pkg/handler/deprecation.go

```diff
@@ -0,0 +1,26 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/MurmurationsNetwork/MurmurationsServices/pkg/jsonapi"
+)
+
+// DeprecationHandler handles requests made to deprecated API versions.
+func DeprecationHandler(c *gin.Context) {
+	errorMessage := "The v1 API has been deprecated. " +
+		"Please use the v2 API instead: " +
+		"https://app.swaggerhub.com/apis-docs/MurmurationsNetwork/LibraryAPI/2.0.0"
```

Comment:
@maxsarratt When deprecation occurs for the Index service, the documentation URL is different, it should be "https://app.swaggerhub.com/apis-docs/MurmurationsNetwork/IndexAPI/2.0.0"

Reply:
Thanks for pointing out!